packages/source-iotsitewise/src/time-series-data/subscribeToTimeSeriesData.ts

```diff
@@ -1,76 +1,70 @@
-import { DescribeAssetModelResponse } from '@aws-sdk/client-iotsitewise';
-import { completeDataStreams } from '../completeDataStreams';
+import { DataModule, DataModuleSubscription, TimeSeriesData, SubscriptionUpdate } from '@iot-app-kit/core';
 import { SiteWiseDataStreamQuery } from './types';
-import { MinimalViewPortConfig } from '@synchro-charts/core';
-import {
-  ErrorDetails,
-  DataModule,
-  DataModuleSubscription,
-  DataStream,
-  TimeSeriesData,
-  SubscriptionUpdate,
-} from '@iot-app-kit/core';
 import { SiteWiseAssetSession } from '../asset-modules';
+import { SiteWiseAlarmModule } from '../alarms/iotevents';
+import { fetchAssetModelsFromQuery } from '../asset-modules/util/fetchAssetModelsFromQuery';
+import { fetchAlarmsFromQuery } from '../alarms/iotevents/util/fetchAlarmsFromQuery';
+import { CreateTimeSeriesDataStore } from './store';
+
+const initialState = {
+  dataStreams: [],
+  annotations: {},
+  assetModels: {},
+  alarms: {},
+  errors: {},
```

Comment:
Do we need to do something with these errors? or are they already captured in the datastreams